build_package.py

```diff
@@ -1606,7 +1633,7 @@ def get_build_info():
             "git config user.email", fail_quietly=True, command_name="git"
         )
         if rc != 0:
-            packager_email = u"unknown"
+            packager_email = "unknown"
```

Comment:
IIRC, This failed in the past if it didn't contain ``u`` prefix.

Reply:
I guess this is the result of the `black` refactorer.
And it also looks like `packager_email` needed only for `print`